libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightDrillConfigPanel/useInsightDrillConfigPanel.ts

```diff
@@ -113,7 +114,10 @@ export const useInsightDrillConfigPanel = (props: IUseDrillConfigPanelProps) =>
         selectAllowMultipleInteractionsPerAttributeAndMeasure,
     );
     const widgetGlobalDrillDowns = useDashboardSelector(
-        selectGlobalDrillsDownAttributeHierarchyByWidgetRef(widgetRef),
+        selectGlobalDrillsDownAttributeHierarchyByWidgetRef(
+            widgetRef,
+            insight?.insight?.properties?.controls?.disableDrillDown,
```

Comment:
Does this selector signature need to be extended? You already have `widgetRef`. Can't you use it inside of the selector to get `insight` and inside of this selector get the prop? You would just change the selector and do not need to update all the other places where it is called.

Reply:
Please check all the selector that you changed signature for if they can work with insight inside of them without changing the signature.

Reply:
I have updated this. Thank you!